### DISCUSSION TITLE
Map terribly slow on startup

### DISCUSSION THREAD

**Discussion by NekiMedo** [osmandapp/OsmAnd#14968] [Q&A]
Anybody else has problems with slow map loading when application is started?
The menu and other interactions work ok, just the map screen updates don't show up i.e. the last map from previous session is shown, and zooming out just shrinks that square with the rest of the screen blank. This goes on for approx. 1 and a half minute and then things are back to normal. Very annoying.
OsmAnd 4.2.7 on Samsung 9, Android 10

**Comment by NekiMedo** [comment]
AN UPDATE
ok, searched the "Issues" tab - looks like disabling visibility (or deleting) recorded tracks fixes this problem, guess i can live with that

**Reply by Boothy99** [reply]
Hi @NekiMedo
I had a similar issue following an update. I discovered it related to having "OpenGL" turned ON. If you're not using 3D view during navigation then you don't need it on.

- Settings
- Go to "Selected Profile" and click on your Current Profile (mine was Walking).
- Scroll down to and click on OsmAnd development
- Make sure "Use OpenGL rendering" is OFF.
- Exit & fully close OsmAnd then re-open.
- See if that fixes your slow starts 👍
- You can show tracks etc again too.

Dave.

**Reply by NekiMedo** [reply]
thanks @Boothy99 (sorry for the late reply got distracted with numerous other issues); checked the OpenGL setting - it was OFF;
at this stage i'm inclined to think those numerous visible recorded tracks were the cause of this issue

**Reply by scaidermern** [reply]
> AN UPDATE ok, searched the "Issues" tab - looks like disabling visibility (or deleting) recorded tracks fixes this problem, guess i can live with that

I wonder if this was the same problem as #14021.
